Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -11,19 +11,61 @@ static void Main(string[] args)
             Test("42*?7=1974", 4);
             Test("42*?47=1974", -1);
             Test("2*12?=247", -1);
-            Console.ReadKey(true);
+            //Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static int FindDigit(String equation){
+            int A,B,C;

```

Comment:
use 'var' rather than specific type for variable declarations wherever possible.  Using 'var' makes your code more readable, using type names makes code little messy when you have large type names, so make it habit of using 'var' for variable declarations whenever possible